zeton/api/child.py

```diff
@@ -53,5 +53,12 @@ def use_points(child_id):
     if used_points > 0:
         if used_points <= current_points:
             zeton.data_access.points.change_points_by(child_id, -used_points, logged_user_id)
+        else:
+            missing_points = abs(current_points - used_points)
+            if missing_points == 1:
+                points = 'punkta'
```

Comment:
Zmieniłbym `points` na `points_word`, gdyż pod `points` spodziewałbym się liczby, ale to moja opinia.